modules/rostests/apitests/ntdll/NtMapViewOfSection.c

```diff
@@ -1818,8 +1818,8 @@ Test_RawSize(ULONG TestNumber)
             ok_hex(ImageFile->zdata_header.SizeOfRawData, RawSizeImageFile.zdata_header.SizeOfRawData);
             ok_hex(ImageFile->zdata_header.PointerToRawData, 0);
 
-            /* PointerToRawData = 0 resets SizeOfRawData to 0, CORE-18797 */
-            ok_hex(ImageFile->bss_header.SizeOfRawData, 0);
+            /* .bss section is unmodified */
```

Comment:
Any explanation then why https://git.reactos.org/?p=reactos.git;a=commit;h=9293b975ef6 was a priori needed?

Reply:
I don't know, but the test was obviously broken.